beets/ui/commands.py

```diff
@@ -684,10 +704,18 @@ def choose_match(self, task):
         # Loop until we have a choice.
         candidates, rec = task.candidates, task.rec
         while True:
+            # Gather extra choices from plugins.
+            # TODO: check for conflicts in choices (duplicated letters, etc)
+            # TODO: provide a way to override summary_judment and/or non-timid
+            # mode? Might be useful for some plugins
+            extra_choices = list(chain(*plugins.send('before_choose_candidate',
```

Comment:
Fully agree (once again)! It should not be too complex to provide a simple way to try solve letter conflicts (loop on the characters of the long string of the choices until a "free" character is found), but seems like just crashing with an informative description should be enough until more plugins start using the event.